SavedInstances/Core/Config.lua

```diff
@@ -198,7 +198,7 @@ function Config:BuildOptions()
         name = L["Show/Hide the SavedInstances tooltip"],
         guiHidden = true,
         type = "execute",
-        func = function() SI:ToggleDetached() end,
+        func = function() SI:GetModule('Tooltip'):ToggleDetached() end,
```

Comment:
instead of calling GetModule every time I would probably just add a new variable at the top